airbyte-bootloader/src/main/java/io/airbyte/bootloader/BootloaderApp.java

```diff
@@ -132,6 +138,14 @@ public BootloaderApp(final Configs configs,
   }
 
   public void load() throws Exception {
+    LOGGER.info("Initializing databases...");
```

Comment:
This class is starting to get fairly confusing to me, because we seem to be creating `Database` objects in both `main()` and in `initPersistences()`, and those are both done before this actual `initialize()` logic is executed. It feels like we should be initializing the databases _before_ we construct the Database objects, right? Lmk if I'm thinking about this incorrectly

Reply:
@lmossman Yes.  I wasn't sure about that.  I think that some of the changes @benmoriceau made introduced that change.  This is all temporary and will be cleaned up when we switch it to use Micronaut.

Reply:
Maybe these `.initialize()` calls should just be moved into the main method before the configs and jobs Database objects are constructed?

But I agree that if this is all temporary then its probably not worth spending too much time refactoring this - as long as this intermediate implementation is still working as expected